### PR TITLE
added the container directive in order to wrap all graphs in a div

### PR DESCRIPTION
**Pull request by getsanjeev**
added the container directive in order to wrap all graphs in a div so that it can be made scroll-able when needed

### CODE REVIEW

**Comment by getsanjeev** [comment]
@pllim @eteq any updates on this? Am I supposed to do anything more in this PR?

**Comment by adrn** [comment]
Hi @getsanjeev! Thanks for this.

Does this work for you locally? I checked out your branch and built the docs locally on my machine, but I don't see any difference between that and the readthedocs version of the docs. Maybe you could post some screenshots with the desired new behavior?

**Comment by Cadair** [comment]
@adrn That's deliberate. The idea is that the sunpy sphinx theme can write css for the container to style it differently. This actually has no effect on astropy. See sunpy/sunpy-sphinx-theme#40

**Comment by adrn** [comment]
@Cadair Got it - that was not clear! 

@getsanjeev In that case, could you please add a comment to the RST explaining why this change was made? Given that it has no impact on the astropy docs, I can imagine Some People (me) being confused when they rediscover this in, say, 2 years :)

**Comment by getsanjeev** [comment]
@adrn @Cadair updated the RST.

**Comment by adrn** [comment]
@getsanjeev Sorry, I should have been more specific. I meant that you should add a comment directly above where you made the change. See inline comment.

File: astropy/coordinates/builtin_frames/__init__.py [hunk 1]
```diff
@@ -90,12 +90,13 @@ def _make_transform_graph_docs():
     frames) is set by the type of transformation; the legend box defines the
     mapping from transform class name to color.
 
+    .. container:: gvcustom
```
Comment (adrn) [inline astropy/coordinates/builtin_frames/__init__.py]: Add a comment (move the one you added to the changelog) to just above this line

Comment (getsanjeev) [reply astropy/coordinates/builtin_frames/__init__.py]: OKay, Will do it.

**Comment by bsipocz** [comment]
superseeded by #7675

**Pull request closed without merging**